azerothcore/azerothcore-wotlk
# fix(DB/page_text_locale): Add some missing zhTW page_text

## Changes Proposed:
-  Add many missing zhTW page_text
- Taken from https://github.com/TrinityCore/TrinityCore/blob/28e962991e56471a80e72510ad629b6f821966c7/sql/updates/world/3.3.5/2022_01_20_00_world.sql (zhCN text was already present)

## Issues Addressed:

- Closes unreported issue.

## SOURCE:

https://github.com/TrinityCore/TrinityCore/blob/28e962991e56471a80e72510ad629b6f821966c7/sql/updates/world/3.3.5/2022_01_20_00_world.sql
https://github.com/leelf00/TrinityCore/blob/0a4e764feae889a8389a2a2259ea835f9246cc10/sql/updates/world/3.3.5/2022_02_06_00_world.sql

## Tests Performed:

![Capture](https://user-images.githubusercontent.com/98835050/152574895-64837120-19e4-46e4-bdf5-38f36db327ca.PNG)



## How to Test the Changes:


1. select * from gameobject where id in (select entry from gameobject_template where type=9 and data0 in (select id from page_text_locale where locale="zhTW"));
2.  Go to any and check page_text using zhTW client

## Known Issues and TODO List:


- There are still many page text that are missing translation. 同志们继续努力！^_^


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
